server/tabby_server/services/resource_format.py

```diff
@@ -71,12 +74,22 @@ def google_books_test_call_api():
     return result_dict.output_dict, HTTPStatus.OK
 
 
+# Generic function for (hopefully) cleaner readability when it comes to
+# retrieving info about books.
+
+def get_book_attributes(book_list, book_attr):
+    try:
+        return book_list["volumeInfo"][book_attr]
+    except KeyError:
+        return {"NA": "Couldn't find!"}
```

Comment:
When the key is not found, I think it would be `None`, as it's the more standard way of demonstrating that no value exists.

Reply:
Fair. I updated to do that.